content/security/docs/multiaccount.md

```diff
@@ -0,0 +1,86 @@
++ More benefits, as described in the [AWS Multi Account Strategy Whitepaper](https://docs.aws.amazon.com/whitepapers/latest/organizing-your-aws-environment/benefits-of-using-multiple-aws-accounts.html#group-workloads-based-on-business-purpose-and-ownership)
+
+The following sections will explain how to implement a multi account strategy for your EKS workloads using either a centralized, or de-centralized EKS cluster approach.
+
+## Planning for a Multi Workload Account Strategy for Multi Tenant Clusters
+
+In a multi account AWS strategy, resources that belong to a given workload such as S3 buckets, ElastiCache clusters and DynamoDB Tables are all created in an AWS account that contains all the resources for that workload. These are referred to as a workload account. Deploying resources into a dedicated workload account is similar to deploying kubernetes resources into a dedicated namespace. 
+
+Workload accounts can then be further broken down by software development lifecycle or other requirements if appropriate. For example a given workload can have a production account, a development account, or accounts for hosting instances of that workload in a specific region. [More information](https://docs.aws.amazon.com/whitepapers/latest/organizing-your-aws-environment/organizing-workload-oriented-ous.html) is available in this AWS whitepaper.
+
+You can adopt the following approaches when implementing EKS Multi account strategy:
+
+## Centralized EKS Cluster
+
+In this approach, your EKS Cluster will be deployed in a single AWS account called the `Cluster Account`. Using [IAM roles for Service Accounts (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) and [AWS Resource Access Manager (RAM)](https://aws.amazon.com/ram/) you can adapt a multi account strategy for your multi tenant EKS cluster. The cluster account will contain the VPC, subnets, EKS cluster, EC2/Fargate compute resources, and any additional networking or configurations needed to run your EKS cluster.
+
+In a multi workload account strategy for multi tenant cluster, AWS accounts typically align with [kubernetes namespaces](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) as a mechanism for isolating groups of resources. [Best practices for tenant isolation](/security/docs/multitenancy/) within an EKS cluster should still be followed when implementing a multi account strategy for multi tenant EKS clusters.
+
+It is possible to have multiple `cluster accounts` in your AWS organization, and it is a best practice to have multiple `cluster accounts` that align with your software development lifecycle needs. For workloads operating at a very large scale, you may require multiple `cluster accounts` to ensure that there are enough kubernetes and AWS service quotas available to all your workloads.
+
+| ![](./images/multi-account-eks.jpg) |
+|:--:|
```

Comment:
We are planning to remove references to this from the best practices guide and our documentation as the service team does not test for it and it isn't guaranteed to work.

Reply:
I meant that you shouldn't try to join nodes from different accounts to a cluster in a different account. The team doesn't test that use case.

Reply:
ack on that - yeah nodes belonging to other accounts isn't a part of our pattern, and I wouldn't recommend it. Can add clarifying notes into the doc that nodes belong to the cluster account.